src/components/Profile/components/Identification/index.tsx

```diff
@@ -745,6 +771,84 @@ const Identification = ({ profile, myProf, isOnline, createSnackbar }: Props) =>
                 ) : null}
 
                 {isOnline && createPhotoDialogBox()}
+
+                {showNameDialog && (
+                  <Dialog
+                    open={showNameDialog}
+                    onClose={() => setShowNameDialog(false)}
+                    className={styles.gc360_links_dialog}
+                    sx={{
+                      '& .MuiDialog-paper': {
+                        minHeight: '300px',
+                        display: 'flex',
+                        flexDirection: 'column',
+                        justifyContent: 'space-between',
+                      },
+                    }}
+                  >
+                    <DialogTitle
+                      sx={{
+                        backgroundColor: 'primary.main',
+                        color: 'primary.contrastText',
+                        padding: '16px 24px',
+                        textAlign: 'center',
+                      }}
+                    >
+                      Name Change Request
+                    </DialogTitle>
+                    <DialogContent
+                      sx={{
+                        display: 'flex',
+                        flexDirection: 'column',
+                        justifyContent: 'center',
+                        flex: 1,
+                        p: 3,
+                      }}
+                    >
+                      <Box
+                        sx={{
+                          backgroundColor: 'action.hover',
+                          borderRadius: 1,
+                          p: 3,
+                          borderLeft: '4px solid',
+                          borderColor: 'secondary.main',
+                          margin: 'auto',
+                          width: '100%',
+                        }}
+                      >
+                        <Typography variant="body2">
+                          <Box component="span" fontWeight="bold">
+                            NOTE:
+                          </Box>
+                          <Box component="ul" sx={{ pl: 2, mt: 1, mb: 0 }}>
+                            <Box component="li" sx={{ mb: 1 }}>
+                              To change your official name, please contact the registrar.
+                            </Box>
+                            <Box component="li" sx={{ mb: 1 }}>
+                              You can contact the registrar via this email:{' '}
+                              <MuiLink
+                                href="mailto:registrar@gordon.edu"
+                                className={styles.gc360_text_link}
+                                underline="hover"
+                              >
+                                registrar@gordon.edu
```

Comment:
These instructions are only applicable to current students. Alumni and facstaff have different processes to change their names (involving contacting the Alumni office and HR, respectively).